### PR TITLE
Feature/adjust secrets and password management for projects

### PR DESCRIPTION
**Pull request by ehrdi**
should not be merged before https://github.com/nanogiants/nanogiants-reusable-actions/pull/9

### CODE REVIEW

File: docs/secrets-and-passwords.md [hunk 1]
```diff
@@ -4,72 +4,126 @@ _A documentation on secure management of secrets and passwords in projects_
...
 
 ## 2. 1Password Vault for Secrets and Passwords
 
 All secrets and passwords of a project must be managed within 1Password. A dedicated vault for each project within 1Password facilitates organized and centralized storage of sensitive information. Access to this vault should be limited to a trusted few, adhering to the principle of "as many as necessary, as few as possible." The use of 1Password ensures that data is encrypted and protected.
 
-### 2.1 Naming Conventions:
+### 2.1 Vault Naming Conventions
 
-To maintain consistency and clarity, we suggest to use the following naming convention for our project vaults: `client-project`. Here's what it means:
+To maintain consistency and clarity, we suggest to use the following naming convention for our project vaults: `client_project`. Here's what it means:
```
Comment (dasheck0) [inline docs/secrets-and-passwords.md:42]: Why did you change from dash case to snake case? Is there a reason to prefer one over the other? I actually like dash case more but this is only a preference. If there is an actual reason why one is better, let's use the better one then.

File: docs/secrets-and-passwords.md [hunk 2]
```diff
@@ -4,72 +4,126 @@ _A documentation on secure management of secrets and passwords in projects_
...
+## 3. 1Password Service Accounts
 
-      - name: Use a secret
-        run: echo ${{ secrets.API_KEY }}
-```
+Service Accounts play a crucial role in enhancing security and optimizing automated processes within our workflows. In the following we outline the process for requesting, storing, and utilizing Service Accounts, emphasizing the importance of maintaining a secure and controlled environment for sensitive information.
+
+### 3.1 Requesting Service Account Access
+
+To request Service Account access, reach out to CTO Stefan N with specific details about the vault requiring access. This request can be submitted through [Fresh Service](https://nanogiantsgmbh.freshservice.com/support/catalog/items/74) or directly via Slack.
```
Comment (dasheck0) [inline docs/secrets-and-passwords.md:71]: Please remove reference to slack ;)

**Review by dasheck0 — CHANGES_REQUESTED** [review]

File: docs/secrets-and-passwords.md (hunk 1 above)
Comment (ehrdi) [reply docs/secrets-and-passwords.md:42]: I like to use kebab-case for names and snake_case for hierarchies. In this case, "client" and "project" are each a level in the hierarchy with their own names. Usually names don't have underlines in them but dashes can occur and that would mess up the hierarchy.
